library/include/rocblas-functions.h

```diff
@@ -15436,6 +15436,12 @@ ROCBLAS_EXPORT rocblas_status rocblas_set_device_memory_size(rocblas_handle hand
  ******************************************************************************/
 ROCBLAS_EXPORT bool rocblas_is_managing_device_memory(rocblas_handle handle);
 
+/*! \brief
+    \details
+    Abort function which safely flushes all IO
+ ******************************************************************************/
+ROCBLAS_EXPORT void rocblas_abort(void) __attribute__((__noreturn__));
```

Comment:
`rocblas_abort` should be added as public function, for people who need to abort, but in a way that's clean to rocBLAS.